ingestion/src/metadata/ingestion/ometa/mixins/es_mixin.py

```diff
@@ -0,0 +1,56 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+Mixin class containing Lineage specific methods
+
+To be used by OpenMetadata class
+"""
+from logging.config import DictConfigurator
+from typing import Generic, TypeVar
+
+from pydantic import BaseModel
+
+from metadata.generated.schema.entity.data.table import Table
+from metadata.ingestion.ometa.client import REST
+from metadata.ingestion.ometa.utils import ometa_logger
+
+logger = ometa_logger()
+
+
+# Prevent sqllineage from modifying the logger config
+def configure(self):
+    pass
+
+
+DictConfigurator.configure = configure
+
+T = TypeVar("T", bound=BaseModel)  # pylint: disable=invalid-name
+
+
+class ESMixin(Generic[T]):
+    client: REST
+
+    es_url: str = (
+        "/search/query?q=service:{} {}&from=0&size=10&index=table_search_index"
```

Comment:
from, size=10 should be parameterized and so is table_search_index. all of these are urls and service:{} AND {}

Reply:
you only 1 result so pass size=1